filter/data.go

```diff
@@ -133,3 +137,20 @@ type Preview struct {
 	NumberOfColumns int        `json:"number_of_columns"`
 	Rows            [][]string `json:"rows"`
 }
+
+type SubmitFilterRequest struct {
+	FilterID       string                    `json:"filter_id"`
+	Dimensions     []filter.DimensionOptions `json:"dimension_options,omitempty"`
+	PopulationType string                    `json:"population_type"`
+}
+
+type SubmitFilterResponse struct {
```

Comment:
You should be aware the main filter response (and I imagine submitFilter will follow suit) response model is about to change to be inline with actual filter-api behaviour (it turns out the swagger spec was very out of date) so you might have to come back and update this.

Reply:
@ChrysmOre dully noted. 
@franmoore05 should this be addressed as part of this ticket?